LightCore.Integration.AspNetCore/Lifecycle/HttpRequestLifecycle.cs

```diff
@@ -10,13 +10,6 @@ namespace LightCore.Integration.AspNetCore.Lifecycle
     /// </summary>
     public class HttpRequestLifecycle : ILifecycle
     {
-        private readonly IHttpContextAccessor _httpContext;
-
-        public HttpRequestLifecycle(IHttpContextAccessor httpContext)
```

Comment:
The IHttpContextAccessor parameter was here to enable Unittesting of the Lifecycle. Why did you remove it?

Reply:
Nope, it wasn't my intent to have this argument to to enable unit testing. I tried to pass in the IHttpContextAccessor at initialization time. That was wrong, because the HttpContext is null. This is why I now create the instance of the HttpContextAccessor directly.
But it would make sense to add an internal constructor with the IHttpContextAccessor as argument, to enable unit testing here.
BTW: This is the ASP.NET Core HttpRequestLifecycle created by me, it is not the ASP.NET HttpRequestLifecycle created by you ;-)

Reply:
Ah okay. Yes you are right :-). The change makes sense now. And the propsal of adding a internal Constructor sounds good!

Reply:
👍

Reply:
done